modules/contentbox/models/ui/ThemeService.cfc

```diff
@@ -664,16 +664,32 @@ component accessors="true" threadSafe singleton{
 							<div class="panel-body">
 					' );
 
+					// Show group intro if there is one
+					if( structKeyExists( thisSettingMD, "groupIntro" ) and thisSettingMD.groupIntro gt '' ){
+						writeOutput( '<div style="padding:5px 0px 5px;border-bottom: 1px solid ##ddd;margin-bottom:20px;">' & thisSettingMD.groupIntro & '</div>' );
+					} 
+					
 					// Set this as the last group
 					lastGroup 	= thisSettingMD.group;
 					firstpanel 	= false;
 				}
 				
 				// writeout control wrapper
-				writeOutput( '<div class="form-group">' );
+				writeOutput( '<div class="form-group" style="margin-top:25px;">' );
 					
 					// write out label
 					writeOutput( html.label( field=settingName, content="#thisSettingMD.label# #requiredText#" ) );
+					
+					// Generate question mark icon for field Help to open modal
+					if( structKeyExists( thisSettingMD, 'fieldHelp' ) && thisSettingMD.fieldHelp gt '' ){
+						writeOutput( ' <a data-toggle="modal" data-target="##help_#settingName#"><i class="fa fa-question-circle"></i></a>' );	
+					}
+					
+					
+					// write out field description
+					if( structKeyExists( thisSettingMD, "fieldDescription" ) and thisSettingMD.fieldDescription gt '' ){
+						writeOutput( '<div style="padding-bottom:5px;">' & thisSettingMD.fieldDescription & '</div>' );
```

Comment:
As above - This was mainly for testing. If we're happy, I added a couple of utility classes and a specific theme class to the css.